ansible/roles/common-cartridge/templates/ingress.yml.j2

```diff
@@ -0,0 +1,24 @@
+#jinja2: trim_blocks: "True", lstrip_blocks: "True"
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ NAMESPACE }}-common-ingress

```

Comment:
Should the naming here be "common-cartridge-ingress"?

Reply:
Done